src/irr/asset/COBJMeshFileLoader.cpp

```diff
@@ -166,101 +182,80 @@ asset::SAssetBundle COBJMeshFileLoader::loadAsset(io::IReadFile* _file, const as
 			break;
 
 		case 'g': // group name
-			{
-				char grp[WORD_BUFFER_LENGTH];
-				bufPtr = goAndCopyNextWord(grp, bufPtr, WORD_BUFFER_LENGTH, bufEnd);
-#ifdef _IRR_DEBUG_OBJ_LOADER_
-	os::Printer::log("Loaded group start",grp, ELL_DEBUG);
-#endif
-				if (ctx.useGroups)
-				{
-					if (0 != grp[0])
-						grpName = grp;
-					else
-						grpName = "default";
-
-                    asset::IAsset::E_TYPE types[] {asset::IAsset::ET_SUB_MESH, (asset::IAsset::E_TYPE)0u };
-                    auto mb_bundle = _override->findCachedAsset(genKeyForMeshBuf(ctx, _file->getFileName().c_str(), mtlName, grpName), types, ctx.inner, 1u).getContents();
-                    if (mb_bundle.first!=mb_bundle.second)
-                    {
-                        auto mb = static_cast<asset::ICPUMeshBuffer*>(mb_bundle.first->get());
-                        mb->grab();
-                        SObjMtl* mtl = findMtl(ctx, mtlName, grpName);
-                        ctx.preloadedSubmeshes.insert(std::make_pair(mtl, mb));
-                    }
-                    else mtlChanged=true;
-
-                    submeshLoadedFromCache = (mb_bundle.first!=mb_bundle.second);
-				}
-			}
```

Comment:
why is this gone?

Reply:
was never used (`if (ctx.useGroups)`, `useGroups` was always false anyway), vertices are divided into submeshes with usemtl tag

Reply:
ok but that controls smoothing etc. AFAIK

Reply:
there is some `s` aka smooth group thing in OBJ spec, but not used/supported in our loader

Reply:
it should be though, I think.

Reply:
should be supported, the smooth groups, I mean

Reply:
i'll have to read more about what this actually is

Reply:
I think its basically which faces/vertices get "artificially" put together for normal recomputation.

This is the whole reason why I wanted the normal recompute function to take a lambda to determine if edge/triangles are connected

Reply:
still a TODO